tests/admin_views/test_autocomplete_view.py

```diff
@@ -61,6 +64,19 @@ def test_success(self):
             'results': [{'id': str(q.pk), 'text': q.question}],
             'pagination': {'more': False},
         })
+        # test for `to_field`
+        fk = ReferencedByParent.objects.create(name='ref by parent')
+        p = ParentWithFK.objects.create(fk=fk)
+        request = self.factory.get(self.url_to_field, {'term': 'by', 'to_field': 'fk'})
```

Comment:
Nothing references `ParentWithFK.fk` so it's a disallowed field. An allowed field would be `ReferencedByParent.name` because it's referenced by `ParentWithFK.fk`.